pkg/skaffold/deploy/resource/deployment.go

```diff
@@ -275,8 +277,10 @@ func (d *Deployment) fetchPods(ctx context.Context) error {
 			case proto.StatusCode_STATUSCHECK_CONTAINER_CREATING,
 				proto.StatusCode_STATUSCHECK_POD_INITIALIZING:
 				event.ResourceStatusCheckEventUpdated(p.String(), p.ActionableError())
+				eventV2.ResourceStatusCheckEventUpdated(p.String(), *sErrors.V2fromV1(p.ActionableError()))
```

Comment:
this `*` is concerning. Can you make `sErrors.V2fromV1` return a value instead of a pointer?